examples/values/backend-org-1.yaml

```diff
@@ -131,3 +131,6 @@ orchestrator:
     - yourchannel:
         restricted: false
         model_export_enabled: true
+
+builder:
+  saveImageTimeoutSeconds: 180
```

Comment:
I'm not sure that was a problem in the CI, was it?

Reply:
Good catch, thanks